### PR TITLE
Require Node.js 4

### PR DESCRIPTION
**Pull request by kevva**
* Add  `ava` and `xo`
* Meta tweaks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
-module.exports = function (str) {
-	var isExtendedLengthPath = /^\\\\\?\\/.test(str);
-	var hasNonAscii = /[^\x00-\x80]+/.test(str);
+module.exports = str => {
+	const isExtendedLengthPath = /^\\\\\?\\/.test(str);
+	const hasNonAscii = /[^\x00-\x80]+/.test(str); // eslint-disable-line no-control-regex
```
Comment (sindresorhus) [inline index.js]: Can you switch those to Unicode escapes? https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-hex-escape.md

File: index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
-module.exports = function (str) {
-	var isExtendedLengthPath = /^\\\\\?\\/.test(str);
-	var hasNonAscii = /[^\x00-\x80]+/.test(str);
+module.exports = str => {
```
Comment (sindresorhus) [inline index.js]: I usually prefer `input` over `str` these days. Can you switch?

File: index.js (hunk 1 above)
Comment (kevva) [inline index.js]: Good idea.

**Comment by sindresorhus** [comment]
Thanks Kevin :)

**Comment by kevva** [comment]
💥

**Pull request merged**